dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/utils/ParameterUtils.java

```diff
@@ -1,133 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.dolphinscheduler.plugin.task.api.utils;
-
-import org.apache.dolphinscheduler.common.constants.DateConstants;
-import org.apache.dolphinscheduler.common.utils.DateUtils;
-import org.apache.dolphinscheduler.plugin.task.api.model.Property;
-import org.apache.dolphinscheduler.plugin.task.api.parser.PlaceholderUtils;
-import org.apache.dolphinscheduler.plugin.task.api.parser.TimePlaceholderUtils;
-
-import org.apache.commons.lang3.StringUtils;
-
-import java.util.Date;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.Map;
-import java.util.Map.Entry;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
-
-/**
- * parameter parse utils
- */
-public class ParameterUtils {
```

Comment:
I would like to keep this under utils rather than move to parser

Reply:
Hi, @ruanwenjun , thanks for you comment

There are 2 `ParameterUtils` and a `ParamUtils` under `task-api` module, and there are many common methods in there 3 classes, such as `convert()`, `dateTemplateParse()`, `convertParameterPlaceholders()`

I think they should be unified in one utils class (which package to put this class in can be discussed), WDYT

Reply:
Keep one class is great, since it is a util, so put under utils is better.

Reply:
> Keep one class is great, since it is a util, so under utils is better.

Sure, I got you